File: homework/03/A10.js
let scores={Alice:85, Bob:92, Charlie:78};
for (let person in scores)
    console.log(`${person} 的分數是 ${scores[person]}`);
/*
`` => 模板字串，可用於書寫多行字串
在模板字串中使用${}來插入變數、表達式或函數調用的結果
for(...in) 用於遍歷物件的屬性或陣列的索引
for(...of) 用於遍歷可迭代物件的每個值
*/

File: homework/08/A10.py
def summary(data):
    for student in data:
        name = student['name']
        scores = student['scores']
        total_score = sum(scores)  # 計算總分
        average_score = total_score / len(scores)  # 計算平均分數
        print(f"學生: {name} | 總分: {total_score} | 平均: {average_score:.2f}")

# 範例資料
students = [
    {'name': 'Alice', 'scores': [90, 80, 70]},
    {'name': 'Bob', 'scores': [100, 85, 95]}
]

# 呼叫函數
summary(students)
